plugins/modules/azure_rm_aks.py

```diff
@@ -938,6 +985,8 @@ def __init__(self):
         self.node_resource_group = None
         self.pod_identity_profile = None
 
+        mutually_exclusive = [('identity','service_principal')]
```

Comment:
missing whitespace after ','
```suggestion
        mutually_exclusive = [('identity', 'service_principal')]
```